dts/bindings/sensor/ite,it8xxx2-vcmp.yaml

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) 2022 ITE Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+description: ITE, it8xxx2 Voltage Comparator node
+
+compatible: "ite,it8xxx2-vcmp"
+
+include: base.yaml
+
+properties:
+  reg:
+    required: true
+
+  label:
+    required: true
+
+  interrupts:
+    required: true
+
+  vcmp-ch:
+    type: int
+    required: true
+    description: Voltage comparator channel.
+
+  scan-period:
+    type: int
+    required: true
+    enum:
+      - 0x10
+      - 0x20
+      - 0x30
+      - 0x40
+      - 0x50
+      - 0x60
+      - 0x70
+      - 0x80
+      - 0x90
+      - 0xa0
+      - 0xb0
+      - 0xc0
+    description: |
+      Select the scan interval for all comparator channel.
+      0x10 = IT8XXX2_VCMP_SCAN_PERIOD_100US, 0x20 = IT8XXX2_VCMP_SCAN_PERIOD_200US,
+      0x30 = IT8XXX2_VCMP_SCAN_PERIOD_400US, 0x40 = IT8XXX2_VCMP_SCAN_PERIOD_600US,
+      0x50 = IT8XXX2_VCMP_SCAN_PERIOD_800US, 0x60 = IT8XXX2_VCMP_SCAN_PERIOD_1MS,
+      0x70 = IT8XXX2_VCMP_SCAN_PERIOD_1_5MS, 0x80 = IT8XXX2_VCMP_SCAN_PERIOD_2MS,
+      0x90 = IT8XXX2_VCMP_SCAN_PERIOD_2_5MS, 0xa0 = IT8XXX2_VCMP_SCAN_PERIOD_3MS,
+      0xb0 = IT8XXX2_VCMP_SCAN_PERIOD_4MS, 0xc0 = IT8XXX2_VCMP_SCAN_PERIOD_5MS
```

Comment:
nit: I'd suggest to provide these as definitions in `include/dt-bindings/...`, so that devicetree will look like:

```
scan-period = <IT8XXX2_VCMP_SCAN_PERIOD_100US>;
```
instead of the less readable
```
scan-period = <0x10>;
```

Reply:
Yes, the definitions is already in `include/dt-bindings/sensor/it8xxx2_vcmp.h` and use the definitions in `boards/riscv/it8xxx2_evb/it8xxx2_evb.dts`.

Reply:
so maybe document this in the binding instead of copying all the values. Like "Check .... file for pre-defined values".

Reply:
Done